### DISCUSSION TITLE
Error: Oh My Zsh can't be loaded from: /bin/sh. You need to run zsh instead.

### DISCUSSION THREAD

**Discussion by Martelinok** [ohmyzsh/ohmyzsh#11084] [Help]
Hi, i have a problem with my Oh My Zsh i have a warning in my terminal when i use Git
![image](https://user-images.githubusercontent.com/9620251/182202367-8fdcb10c-a56d-49a8-8b21-307eae8d196e.png)

**Comment by mcornella** [comment]
Hi, this means that Oh My Zsh was loaded from a non-zsh process. In this case, the process tree indicates that it was loaded as part of the last command:
```
/bin/sh .git/hooks/post-checkout
```
You should inspect the `post-checkout` hook to understand why it was loaded. This could be a `source` line in the `post-checkout` file, or even that you are `source`-ing OMZ inside the `.profile` file.
